plugins/uartdmx/UartWidget.h

```diff
@@ -48,9 +48,9 @@ class UartWidget {
     /**
      * Construct a new UartWidget instance for one widget.
      * @param path The device file path of the serial port
+     * @param padding Minimum DMX frame slots count, padded with NULLs
```

Comment:
Possibly semantics, but I'd say they are zeros here, as they will be actual channel data.